gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/documentation/domain_service/DocumentationValidationDomainService.java

```diff
@@ -50,8 +57,18 @@ public void validateContentIsSafe(String content) {
         }
     }
 
-    public void validateTemplate(String pageContent, String apiId) {
-        this.templateResolverDomainService.resolveTemplate(pageContent, Map.of("api", this.apiCrudService.get(apiId)));
+    public void validateTemplate(String pageContent, String apiId, String organizationId) {
+        var metadata = new HashMap<String, String>();
+        this.apiMetadataQueryService.findApiMetadata(apiId)
+            .forEach((k, v) -> metadata.put(k, v.getValue() != null ? v.getValue() : v.getDefaultValue()));
```

Comment:
I would have used a more functional way to built it by using a `.map` with maybe a `.collect` but 🤷